tests/console/verify_systemd_timesync.pm

```diff
@@ -0,0 +1,55 @@
+# Copying and distribution of this file, with or without modification,
```

Comment:
```
# SUSE's openQA tests
#
# Copyright (C) 2020 SUSE LLC
```
I learn recently that our tests should include this as well. Could you added here?

Reply:
please, pick another test file and paste the whole paragraph instead of adding it or edit it, but good point you mentioned!

Reply:
I must have deleted some line accidentally, thanks for noticing.